src/components/shop/shop-list-screen.js

```diff
@@ -48,7 +49,8 @@ export class ShopList extends Screen {
 
     resize() {
         this.scrollableList.reset();
-        this.balance.resize();
+        this.titles.title.resize();
```

Comment:
Could we bury this in scene and then it would be automatic for every screen?
Does it have it's own onscale hook?

Reply:
can do that 👍

Reply:
Also then you wouldn't need to assign it to the class in the contructor for most applications.